libnmstate/prettystate.py

```diff
@@ -56,11 +56,18 @@ def format_desired_current_state_diff(desired_state, current_state):
 class PrettyState(object):
     def __init__(self, state):
         yaml.add_representer(OrderedDict, represent_ordereddict)
+
+        # unicode does not exist in Python 3. Ignore it there.
```

Comment:
One of the comments got lost: can you use here ‘if six.PY2:’ instead of try/except? It will be more clear and the comment becomes redundant.

Reply:
thx, fixed